tests/integration_tests/security/test_jail.py

```diff
@@ -79,3 +80,15 @@ def test_default_chroot_hierarchy(test_microvm_with_initrd):
                 CHAR_STATS, test_microvm.jailer.uid, test_microvm.jailer.gid)
     check_stats(os.path.join(test_microvm.jailer.chroot_path(),
                              "firecracker"), FILE_STATS, 0, 0)
+
+
+def test_arbitrary_usocket_location(test_microvm_with_initrd):
```

Comment:
Does this need to be `test_microvm_with_initrd`?

Reply:
I get it now, it is kinda confusing to see `initrd` here without knowing this secret :smiley: